src/core/main.lua

```diff
@@ -33,37 +33,34 @@ function main ()
    zone("startup")
    require "lib.lua.strict"
    initialize()
-   local args = parse_command_line()
-   local program = programname(args[1])
-   if program == 'snabb' then
-      -- Print usage with exit status 0 if help requested
-      if args[2] == '-h' or args[2] == '--help' then
-         usage(0)
-      end
-      -- Print usage with exit status 1 if no arguments supplied
-      if #args == 1 then
-         usage(1)
-      end
-      -- Strip 'snabb' and use next argument as program name
-      table.remove(args, 1)
-   end
-   program = select_program(program, args)
+   local program, args = select_program(parse_command_line())
    if not lib.have_module(modulename(program)) then
       print("unsupported program: "..program:gsub("_", "-"))
-      print()
-      print("Rename this executable (cp, mv, ln) to choose a supported program:")
-      print("  snabb "..(require("programs_inc"):gsub("\n", " ")))
-      os.exit(1)
+      usage(1)
```

Comment:
To be more clear, I don't know yet.  However I think removing the redundancy in the names does appear to be a goal; I think if we had named the binary snabb_lwaftr, some upstream code would have already mapped it to program.lwaftr.  So, we'll see how the upstream PR goes.  If there are additional changes to be made there we can merge them in.  I suggest we merge this PR now so as to already have renamed our module tree.